core/src/main/java/com/github/mizool/core/rest/errorhandling/ErrorHandler.java

```diff
@@ -184,30 +174,15 @@ private ErrorResponse handleUndefinedError(Throwable throwable)
 
     private ErrorResponse createUndefinedErrorResponse(Throwable throwable)
     {
-        Map<String, String> parameters = createExceptionParameters(throwable);
-        ErrorDto error = new ErrorDto(ErrorDto.GENERIC_FIELD_KEY, parameters);
-
-        ErrorMessageDto errorMessage = createErrorMessageDto(error);
+        ErrorMessageDto errorMessage = createUndefinedErrorMessageDto(throwable);
         return new ErrorResponse(HttpServletResponse.SC_INTERNAL_SERVER_ERROR, errorMessage);
     }
 
-    private Map<String, String> createExceptionParameters(Throwable throwable)
+    private ErrorMessageDto createUndefinedErrorMessageDto(Throwable throwable)
```

Comment:
most other methods of this class only create `ErrorResponse` instances, and the `ErrorMessageDto` is created by other classes. for consistency, this method should therefore be moved to a separate class, probably called `UndefinedErrorMapper`.